meta/main.yml

```diff
@@ -4,11 +4,11 @@ galaxy_info:
   description: Create or remove local user accounts
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.1
+  min_ansible_version: 2.10
```

Comment:
Maybe then leave this at 2.1 ? As discussed this morning @jburel - I cannot see any dependencies here.

Reply:
I will keep it as it was

Reply:
2.1 does not receive security update. so this should probably be bumped
Following this am discussion I checked the timeline
2.x including 2.10 have reached end of Life

Reply:
> 2.1 does not receive security update. so this should probably be bumped Following this am discussion I checked the timeline 2.x including 2.10 have reached end of Life

There might be a confusion between ansible and ansible-core versions. Unfortunately, when ``ansible --version`` is run, only the ansible-core version is nicely displayed. We are installing ansible in our tests with ``pip install ansible<8`` which installs ansible 7.7.0 (see below)

```
(ansible-supernew) pwalczysko@Petrs-MBP ~ % python --version
Python 3.10.13
(ansible-supernew) pwalczysko@Petrs-MBP ~ % 
(ansible-supernew) pwalczysko@Petrs-MBP ~ % 
(ansible-supernew) pwalczysko@Petrs-MBP ~ % pip install "ansible<8"
Collecting ansible<8
  Downloading ansible-7.7.0-py3-none-any.whl.metadata (7.9 kB)
Collecting ansible-core~=2.14.7 (from ansible<8)
  Downloading ansible_core-2.14.13-py3-none-any.whl.metadata (6.9 kB)
```

So yes, the message in the README is totally flawed and outdated.

Reply:
```

> So yes, the message in the README is totally flawed and outdated.

Although maybe not, it depends on what you call "ansible version". To see the ``ansible version`` which is NOT the ansible-core version, use ``ansible-community``

```
(ansible-supernew) pwalczysko@Petrs-MBP ~ % ansible-community --version
Ansible community version 7.7.0
```

Reply:
Maybe we should specify if we mean the ``ansible-community`` or the ``ansible-core`` versions

https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html

Reply:
Something we probably need to be clear about